adhocracy4/comments_async/static/comments_with_categories/comment_list.jsx

```diff
@@ -23,7 +23,6 @@ export default function (props) {
               content_type={comment.content_type}
               object_pk={comment.object_pk}
               is_deleted={comment.is_deleted}
-              is_moderator_marked={comment.is_moderator_marked}
```

Comment:
We are still using this in spd - did we also remove it elsewhere in a4? Then this can go, otherwise, I think, this might need to stay. :(